travis.cfg

```diff
@@ -1,5 +1,11 @@
 [buildout]
 extends =
-    https://raw.githubusercontent.com/collective/buildout.plonetest/master/travis-4.x.cfg
```

Comment:
Seems that without this line, there is `4.3.x.cfg` as well as `5.1.x.cfg` and `5.2.x.cfg` there is no `test` part being installed and thus tests can not run :sweat_smile: 

Doing a sed replacement here as well for this line would be my suggested change.